docs/renderer.md

```diff
@@ -0,0 +1,61 @@
+---
+name: Renderer
+route: '/renderer'
+---
+
+# Render Engine
+
+## Overview
+
+React requires a rendering engine, typically when creating an application people use `react-dom`.
+When running tests, React still requires an engine. We currently support two different engines –
+`react-test-renderer` & `react-dom`. If you have both installed in your project, by default we will
+use `react-test-renderer`, assuming you're using the default imports:
+
+```js
+import { renderHook } from '@testing-library/react-hooks'
+```
+
+It does this because the library runs a check for available renderers, in the order:
+
+- `react-test-renderer`
+- `react-dom`
+
+If neither are available you will see an error asking you to check that one of the above is
+installed. If only one is installed, then it will use that renderer.
+
+## Being specific
+
+If, however, for certain tests you want to use a specific renderer (e.g. you want to use `react-dom`
+for SSR) you can import the server module directly:
```

Comment:
I like the "Being specific" section, but I think we could have a section (or page?) on SSR where we can talk specifically about hydration and any other server specific quirks.

Thoughts?

Reply:
Doh, it's there, just lower in the file. Ignore this (as although maybe switch the order?)

Reply:
I think SSR you're right, own page. thinking it should go under usage, underneath advanced hooks.